### DISCUSSION TITLE
在ArrayTable场景，检验时只会校验当页数据，如何进行全量校验？

### DISCUSSION THREAD

**Discussion by MisicDemone** [alibaba/formily#2358] [Q&A 2.x]
例：https://codesandbox.io/s/adoring-vaughan-efnwr

点击添加11条数据，然后点提交，翻页器只会标记第一页有校验错误。
如何在提交时校验全量数据呢？

**Comment by janryWang** [comment]
为什么会出现需要全量校验的场景？如果是服务端下发的数据，不应该都是合法数据吗？

**Reply by MisicDemone** [reply]
用户从0条数据添加到很多条，总不能一页一页检查有没有错误吧

**Reply by MisicDemone** [reply]
所有存在翻页的ArrayField场景应该都有这个问题。好像需要创建一些hidden的field用作校验才行？

**Reply by janryWang** [reply]
用户翻页了之后，form.fields就会有字段集了，所以最终提交的时候是肯定会校验的

**Reply by MisicDemone** [reply]
但是目前从demo里看，提交的时候errors只包含了当前页的字段啊

**Reply by janryWang** [reply]
我去，这个的确有bug

**Reply by janryWang** [reply]
这个问题跟#2066 冲突了，目前对unmount后的字段做了跳过校验的处理，得想想怎么解决这个问题了 https://github.com/alibaba/formily/blob/formily_next/packages/core/src/shared/internals.ts#L892

**Reply by j-xzy** [reply]
这样看来unmount后确实不能做跳过校验的逻辑。
